mk/gx.mk

```diff
@@ -2,6 +2,7 @@ gx-path = gx/ipfs/$(shell gx deps find $(1))/$(1)
 
 gx-deps:
 	gx install --global
+	gx-go rw
```

Comment:
I don't think having the build process modify files in place is a very good idea.  The in-place rewrite of the deps in go dependencies already causes me grief in that I have to have automatically reread the files, which doesn't always work.  With go deps, I can at control when it is done, here it will be done as part of `make build`. It can also cause other problems, for example if you have unsaved changes.

Maybe we should copy everything to a temp dir and build and rewrite from that.

Reply:
On the other side, packages on github might not be compatible at the time so you might end up rewriting either way.

Reply:
@Kubuxu I think you misunderstand by idea was to (at least by default) not rewrite deps in place but put the rewritten files into a temp directory and build that.  This way no inplace modifications are necessary.